lib/helpers.js

```diff
@@ -124,18 +124,19 @@ async function runQuery(functionName, queryString, queryParams, pool, clientPara
   }
 }
 
-function getAlertTypeDisplayName(alertType) {
+function getAlertTypeDisplayName(alertType, language, t) {
   let displayName = ''
+
   if (alertType === ALERT_TYPE.SENSOR_DURATION) {
-    displayName = 'Duration'
+    displayName = t(alertType, { lng: language })
   } else if (alertType === ALERT_TYPE.SENSOR_STILLNESS) {
-    displayName = 'Stillness'
+    displayName = t(alertType, { lng: language })
   } else if (alertType === ALERT_TYPE.BUTTONS_NOT_URGENT) {
-    displayName = 'Button Press'
+    displayName = t(alertType, { lng: language })
   } else if (alertType === ALERT_TYPE.BUTTONS_URGENT) {
-    displayName = 'URGENT Button Press'
+    displayName = t(alertType, { lng: language })
   } else {
-    displayName = 'Unknown'
+    displayName = t(alertType, { lng: language })
   }
```

Comment:
Since this is all pretty much the same thing, do you see a way that you can reduce this down to just one line or one if/else?

Reply:
Theresa raises a good point. This group of if statements can be reduced to simply `displayName = t(alertType, { lng: language })`.